### PR TITLE
Fixed #25619 -- Made runserver serve with HTTP 1.1 protocol

### PR DESCRIPTION
**Pull request by claudep**
Thanks Tim Graham for the review.

### CODE REVIEW

File: django/core/servers/basehttp.py [hunk 1]
```diff
@@ -83,13 +84,17 @@ class ThreadedWSGIServer(socketserver.ThreadingMixIn, WSGIServer):
 
 
 class ServerHandler(simple_server.ServerHandler):
+    http_version = "1.1"
```
Comment (timgraham) [inline django/core/servers/basehttp.py]: Use single quotes throughout?

File: django/core/servers/basehttp.py [hunk 2]
```diff
@@ -136,7 +141,15 @@ def get_environ(self):
         return super().get_environ()
 
     def handle(self):
-        """Copy of WSGIRequestHandler, but with different ServerHandler"""
+        """Handle multiple requests if necessary."""
+        self.close_connection = 1
+
```
Comment (timgraham) [inline django/core/servers/basehttp.py]: chop blank line?

File: django/core/servers/basehttp.py [hunk 3]
```diff
@@ -136,7 +141,15 @@ def get_environ(self):
...
+        self.handle_one_request()
+        while not self.close_connection:
+            self.handle_one_request()
+
+    def handle_one_request(self):
+        """Copy of WSGIRequestHandler.handle(), but with different ServerHandler"""
```
Comment (timgraham) [inline django/core/servers/basehttp.py]: comma could be chopped
no blank line after docstring

File: tests/servers/tests.py [hunk 4]
```diff
@@ -50,6 +52,22 @@ def test_live_server_url_is_class_property(self):
 
 
 class LiveServerViews(LiveServerBase):
+    def test_protocol(self):
+        """
+        Ensure launched server serves with HTTP 1.1.
```
Comment (timgraham) [inline tests/servers/tests.py]: no "ensure" . docstring could be a single line version

File: docs/releases/2.0.txt [hunk 5]
```diff
@@ -168,7 +168,7 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* The :djadmin:`runserver` Web server supports HTTP 1.1.
```
Comment (timgraham) [inline docs/releases/2.0.txt]: "Requests and Responses" (I guess), not models.

**Comment by claudep** [comment]
>I'm not sure about this complication. I'd rather investigate switching to gunicorn if possible. Oh well.

AFAIK the main problem is missing Windows support for gunicorn.

>Also, I wonder why Python's server doesn't support multiple connections such that we need a special handle() method.

https://docs.python.org/3.4/library/http.server.html#http.server.BaseHTTPRequestHandler.protocol_version

This specifies the HTTP protocol version used in responses. If set to 'HTTP/1.1', the server will permit HTTP persistent connections; however, your server must then include an accurate Content-Length header (using send_header()) in all of its responses to clients. For backwards compatibility, the setting defaults to 'HTTP/1.0'.

**Pull request merged**